### DISCUSSION TITLE
automerge option for the prs

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [lindell/multi-gitter#317] [General]
Hey,

New to the tool. Did I missed it, but I do not see an option to create the prs with automerge enabled. Such option will be nice.

**Comment by lindell** [comment]
It's currently not possible. But this would definitely be handly for some use-cases. Unfortunately it seems to not be possible through the REST api that multi-gitter uses to create pull requests. It is however possible with the GraphQL API, so if a migration is made this would be easy to implemented. If you are up for it, I would be happy to merge it 

https://stackoverflow.com/a/70867970

**Comment by henri-kohlberg** [comment]
I also raised a [feature request](https://github.com/lindell/multi-gitter/issues/535) for this (not knowing about this discussion) but the problem is still the same: GitHub does not expose the auto-merge flag in their REST API ([documentation](https://docs.github.com/en/rest/pulls/pulls#create-a-pull-request)).
